cylc/flow/scripts/broadcast.py

```diff
@@ -18,34 +18,39 @@
 
 """cylc broadcast [OPTIONS] ARGS
 
-Override or add new [runtime] configuration items in a running workflow.
+Override `[runtime]` configurations in a running workflow.
 
-Uses for broadcast include making temporary changes to task behaviour,
-and task-to-downstream-task communication via environment variables.
+Uses for broadcast include making temporary changes to task behaviour, and
+task-to-downstream-task communication via environment variables.
 
-A broadcast can target any [runtime] namespace (task or task family) for all
-cycles or for a specific cycle. If a task is affected by specific-cycle and
-all-cycle broadcasts at the same time, the specific takes precedence. If a task
-is affected by broadcasts to multiple ancestor namespaces, the result is
-determined by normal [runtime] inheritance. In other words, it follows this
-order:
+See also `cylc reload` which reads in the flow.cylc (or suite.rc) file.
 
+A broadcast can set/override any `[runtime]` configuration for all cycles or
+for a specific cycle. If a task is affected by a specific-cycle and an
+all-cycle broadcasts at the same time, the specific takes precedence.
+
+Broadcasts can also target all tasks, specific tasks or families of tasks. If a
+task is affected by broadcasts to multiple ancestor namespaces (tasks it
+inherits from), the result is determined by normal `[runtime]` inheritance. In
+other words, it follows this order:
+
+```
 all:root -> all:FAM -> all:task -> tag:root -> tag:FAM -> tag:task
+```
```

Comment:
This is for the CLI so markdown doesn't work
```suggestion
all:root -> all:FAM -> all:task -> tag:root -> tag:FAM -> tag:task
```